src/phoenix/server/api/dataloaders/min_start_or_max_end_times.py

```diff
@@ -28,6 +30,16 @@
 ResultPosition: TypeAlias = int
 DEFAULT_VALUE: Result = None
 
+_Section = ProjectRowId
+_SubKey = Kind
+
+
+class MinStartOrMaxEndTimeCache(
+    TwoTierCache[Key, Result, _Section, _SubKey],
+    main_cache_factory=lambda: LFUCache(maxsize=16),
+    sub_cache_factory=lambda: LFUCache(maxsize=2),
+): ...
```

Comment:
Not clear what work this one is doing?

Reply:
It's just instantiating the class object (which for the purpose of the cache is basically singleton, but we don't really make it a singleton)

Reply:
this is to set the eviction policy, i.e. LFU